### PR TITLE
Improvements to replication conceptual docs

### PR DESCRIPTION
**Pull request by ravindk89**
# Summary

Improvements to site replication conceptual docs w.r.t. availability/resiliency and architecture.

Documenting proxy behavior on GET/HEAD

Cleanups of images plus additional improvements after Oscar's initial work.

# Staged:

- http://192.241.195.202:9000/staging/replconcept/linux/operations/concepts/architecture.html#replicated-minio-deployments
- http://192.241.195.202:9000/staging/replconcept/linux/operations/concepts/availability-and-resiliency.html#replicated-minio-deployments
- http://192.241.195.202:9000/staging/replconcept/linux/operations/install-deploy-manage/multi-site-replication.html#proxy-to-other-sites

### CODE REVIEW

**Comment by ravindk89** [comment]
@oscarocastellanos I modified some of your graphics, LMK what you think or feel free to edit and suggest improvements.

**Comment by oscarocastellanos** [comment]
@ravindk89 Everything looks great. Just did a tiny quick edit at `architecture-load-balancer-multi-site.svg` because there was an arrow floating over there. But overall, great work on these.
architecture-load-balancer-multi-site

File: source/operations/concepts/availability-and-resiliency.rst [hunk 1]
```diff
@@ -158,4 +161,48 @@ For multi-pool MinIO deployments, each pool requires at least one erasure set ma
...
+      The Load Balancer handles routing operations to Site A in Datacenter 1.
+      Site A continuously replicates data to Site B.
+
+   Once all data synchronizes, you can restore normal connectivity to that site.
+   Depending on the amount of replication lag, latency between sites and overall workload :abbr:`I/O (Input / Output)`, you may need to temporarily stop write operations to allow the sites to completely catch up.
+
+   If a peer site completely fails, you can remove that site from the configuration entirely.
+   The load balancer configuration should also remove that site to avoid routing client requests to the offline site.
+
+   You can then restore the peer site, either after repairing the original hardware or replacing it entirely, by adding it back to the site replication configuration.
```
Comment (djwfyi) [inline source/operations/concepts/availability-and-resiliency.rst:190]: Is this expanding the site replication? I know there were issues initially with this, but I can't remember what the end of the fixes was.

It's worth linking the last last phrase to one of the tutorials. Maybe https://min.io/docs/minio/linux/operations/install-deploy-manage/multi-site-replication.html#expand-site-replication.

File: source/operations/install-deploy-manage/multi-site-replication.rst [hunk 2]
```diff
@@ -88,6 +88,23 @@ Any MinIO deployment in the site replication configuration can resynchronize dam
...
+
+1. A client issues ``GET("data/invoices/january.xls")`` to ``Site1``
+2. ``Site1`` cannot locate the object
+3. ``Site1`` proxies the request to ``Site2``
+4. ``Site2`` returns the latest version of the requested object
+5. ``Site1`` returns the proxied object to the client
+
+For ``GET/HEAD`` requests that do *not* include a unique version ID, the proxy request returns the *latest* version of that object on the peer site.
+This may result in retrieval of a non-current version of an object, such as if the responding peer site is also experiencing replication lag.
+
```
Comment (djwfyi) [inline source/operations/install-deploy-manage/multi-site-replication.rst:107]: Might be worth reiterating here that this is not true for `LIST` operations. They must present to a healthy peer site.

**Review by djwfyi — APPROVED** [review]
Two small suggestions, one to add a link, and one to add a note about the LIST behavior with proxying requests.

Otherwise, LGTM.

**Comment by feorlen** [comment]
Not quite sure what this is about, but if I open some images in a new browser tab, they are mangled?

Screenshot of two browser windows, one showing the image in the document and the other by itself in a new window. The new window image has oddly scrunched up text.

The reason I tried this in the first place is that some of the text is too small for me to read.

**Comment by oscarocastellanos** [comment]
I think these kept the fonts. We need to convert the text to outlnes instead of text @feorlen

**Review by poornas — APPROVED** [review]

**Comment by ravindk89** [comment]
> I think these kept the fonts. We need to convert the text to outlnes instead of text @feorlen

Let me see if I can do this...I just have to convert everything into paths right?

**Comment by ravindk89** [comment]
> Not quite sure what this is about, but if I open some images in a new browser tab, they are mangled?
> 
> Screenshot of two browser windows, one showing the image in the document and the other by itself in a new window. The new window image has oddly scrunched up text.
> 
> The reason I tried this in the first place is that some of the text is too small for me to read.
> 
> Browser-dependent, apparently. Brave doesn't do that. But opening the image in a new window doesn't make it any larger.

Hm - so I'm converting these all to paths, but that wont solve the _text too small_ issue. For that we would have to not use SVGs and use PNGs only. Even then I'm not sure if the "lightbox" would work.

@rushenn would it be possible/difficult to have some sort of JS that would let you click an SVG and view it in a larger box?

File: source/operations/install-deploy-manage/multi-site-replication.rst (hunk 2 above)
Comment (ravindk89) [reply source/operations/install-deploy-manage/multi-site-replication.rst:107]: True for PUT and Delete too. Adding that.

**Pull request merged**